### DISCUSSION TITLE
SolaX sensor

### DISCUSSION THREAD

**Discussion by Fuglebakken19** [wills106/homeassistant-solax-modbus#451] [Q&A] [answered]
I have the SolaX Inverter Modbus installed, using my Ginlong Solis inverter. 
I'm trying to find the sensor in SolaX which shows both import and export from the grid. A sensor that goes both positive and negative depending if it's importing or exporting to the grid. 

Can someone tell me what it's called ? Cuz I can't seem to find it. 

Thanks in advance! 🙏🏻

**Comment by alienatedsec** [comment]
@Fuglebakken19 
![image](https://github.com/wills106/homeassistant-solax-modbus/assets/73167064/69d80848-4f6e-4810-a280-729997ea6aa8)

**Reply by Fuglebakken19** [reply]
Thank you!

**Reply by Fuglebakken19** [reply]
@alienatedsec Can you also let me know the sensor that reads the import power from the grid in Watts ?

🙏🏻

**Reply by alienatedsec** [reply, marked as answer]
@Fuglebakken19 the above is import power in watts - whenever it goes to minus it imports from the grid, whenever it goes to plus it exports to the grid

**Reply by Fuglebakken19** [reply]
Thank you very much 👍
